### PR TITLE
T 13697 nft.trades/mints/burns/transactions/fees rework

### PR DESCRIPTION
**Pull request by soispoke**
Brief comments on the purpose of your changes:

*For Dune Engine V2*
I've checked that:

* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`

When you are ready for a review, tag duneanalytics/data-experience. We will re-open your forked pull request as an internal pull request. Then your spells will run in dbt and the logs will be avaiable in Github Actions DBT Slim CI. This job will only run the models and tests changed by your PR compared to the production project.

### CODE REVIEW

File: spellbook/models/nft/nft_schema.yml [hunk 1]
```diff
@@ -14,21 +14,99 @@ models:
       - &blockchain
         name: blockchain
         description: "Blockchain"
-      - name: project
+      - &project
+        name: project
         description: "Project"
```
Comment (dot2dotseurat) [inline spellbook/models/nft/nft_schema.yml:19]: personally, I think if the description is the same as the column name, we can just omit. unless we are adding some detail, it's just duplicative.

File: spellbook/models/nft/nft_trades.sql [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
 
-SELECT blockchain, project, version, tx_hash, block_time, amount_usd, amount, token_symbol, token_address, unique_trade_id FROM 
-(SELECT blockchain, project, version, tx_hash, block_time, amount_usd, amount, token_symbol, token_address, unique_trade_id FROM {{ ref('opensea_trades') }} 
-UNION ALL
-SELECT blockchain, project, version, tx_hash, block_time, amount_usd, amount, token_symbol, token_address, unique_trade_id FROM {{ ref('magiceden_trades') }}) 
+SELECT *
+FROM 
+(
+SELECT * FROM {{ ref('opensea_trades') }} 
+        UNION ALL
```
Comment (dot2dotseurat) [inline spellbook/models/nft/nft_trades.sql:11]: reminder. I think this should be union not union all. just calling out so it doesn't slip through.

File: spellbook/models/transfers/ethereum/erc20/transfers_ethereum_erc20.sql [hunk 3]
```diff
@@ -3,7 +3,7 @@
 with
     sent_transfers as (
         select
-            'send' || '-' || evt_tx_hash || '-' || evt_index || '-' || `to` as unique_tx_id,
+            'send' || '-' || evt_tx_hash || '-' || evt_index || '-' || `to` as unique_transfer_id,
```
[the remark this answers is not in the document]
Comment (soispoke) [reply spellbook/models/transfers/ethereum/erc20/transfers_ethereum_erc20.sql]: Yes, done :)

File: spellbook/models/magiceden/magiceden_trades.sql [hunk 4]
```diff
@@ -7,12 +7,40 @@
...
+amount_raw,
+currency_symbol,
+currency_contract,
+NULL::string as currency_contract_original,
+NULL::string as nft_contract_address,
+project_contract_address,
+NULL::string as aggregator_name,
+NULL::string as aggregator_address,
+tx_hash,
+NULL::string as tx_from,
```
[the remark this answers is not in the document]
Comment (soispoke) [reply spellbook/models/magiceden/magiceden_trades.sql:30]: No definitely not, but we need to spend some time finding how to add these fields for Solana NFT trades (I did work on it quite a bit already but I want to leave it for another PR). The goal is to have everyone of these fields for Solana nft trades as well

File: spellbook/models/nft/nft_schema.yml (hunk 1 above)
Comment (soispoke) [reply spellbook/models/nft/nft_schema.yml:19]: Agreed

File: spellbook/models/nft/nft_trades.sql (hunk 2 above)
Comment (soispoke) [reply spellbook/models/nft/nft_trades.sql]: Done :) Thanks for your review !

**Pull request closed without merging**